courses/machine_learning/deepdive2/art_and_science_of_ml/solutions/neural_network.ipynb

```diff
@@ -36,10 +36,16 @@
     }
```

Comment:
<p>Line #5, changes Tensorflow to TensorFlow.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>